README.md

```diff
@@ -0,0 +1,9 @@
+# Multi-purpose Authentication Site
+Multi-purpose Authentication Site is Identity Provider (Idp) and Security Token Service (STS) of OAuth 2.0 and OpenID Connect powered by ASP.NET Identity and JSON Web Token (JWT).
```

Comment:
@daisukenishino 

See [this comment](https://github.com/OpenTouryoProject/MultiPurposeAuthSite/pull/46#discussion_r139873352).